fsd_utils/mapping/application/qa_mapping.py

```diff
@@ -10,7 +10,7 @@
 from fsd_utils.mapping.application.multi_input import MultiInput
 
 
-def extract_questions_and_answers(forms) -> dict:
+def extract_questions_and_answers(forms, language) -> dict:
```

Comment:
can you set the default `language` value to `en` ?

Reply:
have done this now